### DISCUSSION TITLE
Don't scroll focused tab into view

### DISCUSSION THREAD

**Discussion by just-mitch** [tailwindlabs/headlessui#939] [Help]
I'm using the product photo gallery, as shown in tailwind UI here:

https://tailwindui.com/components/ecommerce/components/product-overviews#component-13a89c2dc50a31afd66541dc28fa3c13

However, in my case, there can be many photos, and when I hit the arrow key to move to the next photo, the browser scrolls to the next thumbnail, since it came into focus, but I would like it to not scroll. 

Is there a workaround for this?

Thanks so much!

**Comment by emrecoban** [comment]
The same with me! the `overflow-x-scroll` doesn't work unfortunately. Is it not responsive?

[HeadlessUI Tabs](https://headlessui.com/react/tabs)
